spec.bs

```diff
@@ -888,9 +890,9 @@ A <dfn for=fencedframetype>destination event</dfn> is either a
         1. Otherwise, set |attributionReportingEligibility| to "<code>[=eligibility/event-source=]</code>".
 
         1. Set |processResponse| to these steps given a [=response=] |response|:
-
+           1. Let |fenced| be true.
            1. Run [=process an attribution eligible response=] with |event|'s
-              [=destination enum event/attributionReportingContextOrigin=], |attributionReportingEligibility|, and |response|.
+              [=destination enum event/attributionReportingContextOrigin=], |attributionReportingEligibility|, |fenced|, |request|'s [=request/trigger verification metadata=], and |response|.
```

Comment:
Can we just pass in true? Looks like we don't need the extra `|fenced|` variable?

Reply:
yeah we can, but just thought it's clearer what this parameter is. Wdyt?

Reply:
I see, so it is basically self-documenting code like `/*parameter_foo=*/true` would be. Hmm, OK I'm fine with it.